.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: required
+sudo: false
```

Comment:
@cauealves done :+1:
On Jun 18, 2016 12:14 AM, "Cauê Alves" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/cauealves/pagespeed-score-cli/pull/8#discussion_r67557294>
> :
>
> > @@ -1,4 +1,4 @@
> > -sudo: required
> > +sudo: false
>
> @tarungarg546 <https://github.com/tarungarg546> I put to perform a test
> and it was my fault not to remove it.. You can remove this 👍
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cauealves/pagespeed-score-cli/pull/8/files/7596f890a15d366fdb95adfec6a26efe283fb991#r67557294>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ86icwfxL9RrLzNQaDNbqsTPiCqPT0lks5qMusHgaJpZM4I3w2i>
> .
>